app/workers/immediate_email_generation_worker.rb

```diff
@@ -0,0 +1,7 @@
+class ImmediateEmailGenerationWorker
+  include Sidekiq::Worker
+
+  def perform
+    ImmediateEmailGenerationService.call
```

Comment:
Not something new here, but I wonder whether we might want to ditch that service and just have a fatter worker? Maybe I'm missing what we're gaining.

Reply:
Agreed - and I'll rebase this off #379 and do the same for that.